build.gradle

```diff
@@ -8,18 +8,14 @@ plugins {
 archivesBaseName = 'taskwarrior-java-client'
 description = 'A Java client to communicate with a taskwarrior server (= taskd).'
 group = 'de.aaschmid'
-version = '1.0-SNAPSHOT'
+version = '1.1-SNAPSHOT'
 
 sourceCompatibility = targetCompatibility = JavaVersion.VERSION_1_8
 
 repositories {
     mavenCentral()
 }
 
-task wrapper(type: Wrapper) {
```

Comment:
my intellij idea barks on conflicting task. Removing this does not seem to break anything.

Reply:
with new versions of Gradle this `task` is obsolete. So you can safely remove it. 
And if you like you can upgrade to the newest Gradle wrapper...